insights/client/support.py

```diff
@@ -53,8 +48,7 @@ def _legacy_registration_check(pconn):
 
     return {'messages': [local_record, api_record],
             'status': api_reg_status,
-            'unreg_date': unreg_date,
-            'unreachable': unreachable}
+            'unreg_date': unreg_date}
 
 
 def registration_check(pconn):
```

Comment:
Do I understand it correctly that the modern API does not track unregistered systems? It looks like [_insights.client.connection.InsightsConnection.api_registration_check_](https://github.com/RedHatInsights/insights-core/blob/bb04fb85035fbb0eb8bab72486328bb2ad8cc9d1/insights/client/connection.py#L729) [does not distinguish](https://github.com/RedHatInsights/insights-core/blob/bb04fb85035fbb0eb8bab72486328bb2ad8cc9d1/insights/client/connection.py#L743) between not found and unregistered if _legacy_upload_ is _False_.